regression/memory-analyzer/pointer_08/test.desc

```diff
@@ -0,0 +1,5 @@
+KNOWNBUG
```

Comment:
Same comment as `FUTURE`.

Reply:
The requested symbol has type `void*`. Explanation added.